keps/7610-default-local-queue-creation/kep.yaml

```diff
@@ -0,0 +1,37 @@
+title: Create Default LocalQueues Based on NamespaceSelector
+kep-number: 7610
+authors:
+  - "@j-skiba"
+status: provisional
+creation-date: 2025-11-12
+reviewers:
+  - TBD
+approvers:
+  - TBD
+
+see-also: []
```

Comment:
add reference to the localqueue-defulting kep
plus also add the related issue in LWS